reason-react-native/src/apis/Style.md

```diff
@@ -4,4 +4,1298 @@ title: Style
+in this random examples
+
+- `~height=10.5->pt` (= `~height=pt(10.5)`)
+- `~width=55.->pct` (= `~width=pct(55.)`)
+- `~margin=auto`
+
+### `offset`
+
+`offset` is used for `shadowOffset` & `textShadowOffset`.
+
+Eg: `~shadowOffset=offset(~height=2., ~width=4.)`
+
+### `angle`
+
+`angle` is used for transforms. It can be expressed in degrees (`deg` function)
+or in radians (`rad` function):
+
+- `20.->deg` (eg: `rotateX(~rotateX=20.->deg)`
+- `0.5.->rad` (eg: `rotateZ(~rotateZ=0.5->rad)`
+
+## Style Functions
+
+### `Style.style`
+
+For convenience, this function allow you to prepare a style object with all
+styles available in React Native. The nice side of this is that you can use this
+function & not think about what component is going to use it. On the other hand,
+this function can trigger React Native error screen (eg: if you pass `color` to
+a `View` component).
+
+`style` accepts all the Style Props:
+
+- [Layout styles](#layout-styles-props)
+- [Shadow styles](#shadow-styles-props)
+- [Transform styles](#transform-styles-props)
+- [View styles](#view-styles-props)
+- [Text styles](#text-styles-props)
+- [Image styles](#image-styles-props)
+
+⚠️ _Internally, React Native codebase types for this function are called
+`____DangerouslyImpreciseStyle_Internal`. That's why we created new functions:
+[`viewStyle`](#style-viewstyle), [`textStyle`](#style-textstyle) &
+[`imageStyle`](#style-imagestyle)_.
+
+### `Style.viewStyle`
+
+This function accept all React Native following styles
+
+- [Layout styles](#layout-styles-props)
+- [Shadow styles](#shadow-styles-props)
+- [Transform styles](#transform-styles-props)
+- [View styles](#view-styles-props)
+
+### `Style.textStyle`
+
+This function accept all React Native following styles
+
+- [Layout styles](#layout-styles-props)
+- [Shadow styles](#shadow-styles-props)
+- [Transform styles](#transform-styles-props)
+- [Text styles](#text-styles-props)
+
+### `Style.imageStyle`
+
+This function accept all React Native following styles
+
+- [Layout styles](#layout-styles-props)
+- [Shadow styles](#shad@ow-styles-props)
+- [Transform styles](#transform-styles-props)
+- [Image styles](#image-styles-props)
+
+## Style Props
+
+### Layout Style Props
+
+[Official documentation](https://facebook.github.io/react-native/docs/layout-props)
+
+#### `alignContent`
+
+Accept one of the following values:
+
+- `` `flexStart ``
+- `` `flexEnd ``
+- `` `center ``
+- `` `stretch ``
+- `` `spaceAround ``
+- `` `spaceBetween ``
+
+Controls how rows align in the cross direction, overriding the `alignContent` of
+the parent.
+
+- [Web reference](https://developer.mozilla.org/en-US/docs/Web/CSS/align-content)
+- [Yoga reference](https://yogalayout.com/docs/align-content/)
+
+#### `alignItems`
+
+Accept one of the following values:
+
+- `` `flexStart ``
+- `` `flexEnd ``
+- `` `center ``
+- `` `stretch ``
+- `` `baseline ``
+
+Default ``stretch`.
```

Comment:
typo:
`` `stretch ``

Perhaps you could state the default within the list, as in
- `` `stretch `` (default)

Reply:
Done everywhere where I know the default value.